### DISCUSSION TITLE
Is there a way to manage Oauth without a netifly account?

### DISCUSSION THREAD

**Discussion by aloxe** [sveltia/sveltia-cms#196] [Q&A] [answered]
I have a 11ty site that has it own hosting server. The repo and the CI is on github.

Ideally I would just need to add sveltia to site. When I run on localhost, It can authenticate directly with my github user but when I deploy it, The admin page wants to look at https://api.netlify.com/auth?provider=github&site_id=mydomain.example.com&scope=repo which is a page not found.

Is there a way to identify like on localhost or is it needed it have a netifly account just for this?

**Comment by kyoshino** [comment, marked as answer]
Netlify is used by default for compatibility with Netlify/Decap CMS. If you don’t (want to) have a Netlify account, you can use another OAuth client. As mentioned in the [README](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#moving-your-site-from-netlify-to-another-hosting-service), we actually have [our own](https://github.com/sveltia/sveltia-cms-auth), so you can give it a try! Once you’ve set it up, you’ll be able to sign in with GitHub remotely 🙂

**Reply by aloxe** [reply]
Thanks for the answer, 
I finally had to host my own one. I used [netlify-cms-oauth-provider-php](https://github.com/mcdeck/netlify-cms-oauth-provider-php) with the documentation in this [blog post](https://www.van-porten.de/blog/2021/01/netlify-auth-provider/).
The key point is to add `base_url: https://myoauth.example.org` in the config.xml
